extensions/functions_rounding.yaml

```diff
@@ -0,0 +1,27 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "ceil"
+    description: "Rounding to the ceiling of the value"
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
```

Comment:
ceil still has overflow options (like floor, it doesn't need any options AFAICT).